.template.config/template.json

```diff
@@ -6,7 +6,7 @@
     "WebAPI",
     "CQRS"
   ],
-  "name": "Amido Stacks Web Api CQRS - Full solution",
+  "name": "Amido Stacks Web Api CQRS",
```

Comment:
It should be `Web API` since that's the way Microsoft list their api templates

Reply:
Changed to "Amido Stacks Web API CQRS"